### PR TITLE
add clarification about append-only mode, fixes #1689

### LINKED ISSUE #1689: append_only and prune interaction
Is it safe to automatically run prune (from the safe client) on append_only repository?

Is there a protection from the following situation:

1) Hacked client A with append_only access iterates over chunk id's from the cache and put() garbage to them.
2) Client B with full access runs a prune command.

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -800,9 +800,14 @@ That's all to it.
 Drawbacks
 +++++++++
 
-As data is only appended, and nothing deleted, commands like ``prune`` or ``delete``
+As data is only appended, and nothing removed, commands like ``prune`` or ``delete``
 won't free disk space, they merely tag data as deleted in a new transaction.
 
+Be aware that as soon as you write to the repo in non-append-only mode (e.g. prune,
+delete or add archives from a admin machine), it will remove the deleted objects
```
Comment (enkore) [inline docs/usage.rst]: from an admin

File: docs/usage.rst [hunk 2]
```diff
@@ -800,9 +800,14 @@ That's all to it.
...
+including the ones that were already marked as deleted, but not removed, in append-only
+mode.
```
Comment (enkore) [inline docs/usage.rst]: remove ... objects ... [permanently]?

File: docs/usage.rst (hunk 1 above)
Comment (ThomasWaldmann) [inline docs/usage.rst]: ok

File: docs/usage.rst (hunk 2 above)
Comment (ThomasWaldmann) [inline docs/usage.rst]: ok

**Pull request merged**